tasks/skylab/StarAlign.wdl

```diff
@@ -266,14 +267,19 @@ task STARsoloFastq {
 
 
     COUNTING_MODE=""
-    if [ "~{counting_mode}" == "sc_rna" ]
+    if [[ "~{counting_mode}" == "sc_rna" ]]
```

Comment:
@khajoue2 What happens if someone runs sc_rna mode with the exon_counts as true? If we don't want them to do that, perhaps we could put a comment in or return an error?

Reply:
Added a check to not remake the wdl.